todo_app/data/mongo_db_client.py

```diff
@@ -0,0 +1,66 @@
+import datetime
+from bson import ObjectId
+from flask import current_app
+import pymongo
+from todo_app.data.item import Item
+import todo_app.data.configs.logger_settings as logger
+import os
+
+class MongoDBClient:
+
+    def __init__(self):
+        self.client = pymongo.MongoClient(os.getenv('MONGO_CON_STRING'))
+        self.db = self.client[os.getenv('MONGO_DB_NAME')]
+        self.items = self.db.items
+
+    def get_items(self):
+        """
+        Get all items from the Mongo DB.
+
+        Returns:
+            items: List of items' as dictionary.
+        """
+        items = list(self.items.find())
+        cards = []
+        for card in items:
+            cards.append(Item(
+                card['_id'],
+                card['name'],
+                card['description'] if 'description' in card else None,
+                card['due'] if 'due' in card else None,
```

Comment:
You accept a "due" field here but it looks like it's impossible to set that in the current version of the app - if that's right then we might as well get rid of this from our class model!

Reply:
It's working locally and on heroku for me. Here's the route:

@app.route('/item_due/<string:titleID>', methods=['GET', 'POST'])
    def item_due(titleID):
        due = request.form['date']
        ...
        mongo_db.update_document(titleID, 'due', due)
        ...
        
Are you facing issues with using the calendar and update button on your end?